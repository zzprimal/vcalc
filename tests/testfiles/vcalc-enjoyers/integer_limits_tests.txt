int var123 = (0 - 2147483647) / 100000;
print(var123);
vector var = 1..214740364;
int var124 = 2147483647;
print(var[2147483] - var124);
print(0 - var124);
vector l = 1..60;
print(l[var[2147483]] + l[var[21]] + l[var[2147]] - l[var[0 - 1]] + l[var[0 - 2147483647 - 1]]);
//CHECK_FILE:./integer_limits_tests.out