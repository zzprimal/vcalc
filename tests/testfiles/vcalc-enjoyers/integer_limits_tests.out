-21474
-2145336163
-2147483647
23
